### ISSUE TITLE
Error on Staking Payouts Page for Polkadot Local Testnet

### ISSUE THREAD

**Issue by jack-kearney** [polkadot-js/apps#2758]
I've been using the Polkadot Local Testnet (`--dev`) while developing recently and have noticed that the `#/staking/payouts` page fails to render due to the following error:

```
Uncaught error. Something went wrong with the query and rendering of this component. Cannot read property 'batch' of undefined
```

This page renders properly when querying a Kusama node.

If the payout functionality is somehow not included or possible in the local testnet, it would be great to display that information instead of causing an error. Otherwise, this is a bug in the application - fixing this would allow me (and other folks running validators) to better develop our payout & accounting logic prior to mainnet launch.

Please let me know if I can clarify this issue further. Thanks!

**Comment by jacogr** [comment]
Sadly, there is not extra information that "comes back". This is caused by type mismatches - the teams are still heavily refactoring weights, calls & types so the permutations are a lot. 

Will take a look when possible, at this point polkadot master does not work/compile.

(It does work with substrate master and dev, which is the version that will be included on live/mapped to polkadot master, so it s a matter of updating Polkadot with the adjusted types as found - https://github.com/polkadot-js/api/blob/master/packages/types-known/src/spec/polkadot.ts)

TL;DR These are all rapidly moving targets, there are dragons.
